tests/test_tartiflette_app.py

```diff
@@ -13,3 +15,24 @@ def test_path(engine: Engine):
 
     assert response.status_code == 200
     assert response.json() == {"data": {"hello": "Hello stranger"}}
+
+
+def test_tartiflette_app_as_sub_starlette_app(engine: Engine):
+
+    async def home(request):
+        return PlainTextResponse("Hello, world!")
+
+    graphql = TartifletteApp(engine=engine)
+    routes = [
+        Route("/", endpoint=home),
+        Mount("/graphql", app=graphql, name="tartiflette-asgi")
+    ]
+    app = Starlette(routes=routes, on_startup=graphql.startup)
```

Comment:
on_startup should be a list https://github.com/encode/starlette/blob/master/starlette/applications.py#L46

I'm wondering why test passed :thinking: